doc-site/components/nve-alert.md

```diff
@@ -1,127 +1,117 @@
 ---
 layout: component
+outline: [2, 3]
 ---
 
 <CodeExamplePreview>
 
 ```html
-<nve-alert open text="text"></nve-alert>
+<nve-alert open label="Info" text="Dette er alert komponent. Den har ventet hele livet på å varsle deg!"></nve-alert>
 ```
 
 </CodeExamplePreview>
 
-<nve-message-card label="Tips">
-Bruk `open` for å vise en alert. Hvis ikke `open` er satt, vises den ikke.
+<nve-message-card label="Tips" size="compact">
+<p>Bruk <b>open</b> for å vise en alert. Hvis ikke <b>open</b> er satt, vises den ikke.</p>
 </nve-message-card>
 
 ## Eksempler
 
-### Variant
+### Styr innhold
 
-Bruk `variant` for å styre farger. `primary` er standard og trenger ikke spesifiseres.
+Du kan bruke spor for å vise innhold i alerten. Standard-spor er hovedteksten. Du kan også bruke egne spor for ikon (icon) og overskrift (label).
 
-<CodeExamplePreview arrangeComponentsVertically>
+<CodeExamplePreview>
 
 ```html
 <nve-alert open>
   <nve-icon slot="icon" name="info"></nve-icon>
```

Comment:
Vil ikke dette vises uansett om denne linja er med eller ikke?
Hva om du brukte et annet ikon for å demonstrere at det funker?